app/views/users/_posts.html.erb

```diff
@@ -0,0 +1,5 @@
+ <div class="flex items-center justify-center mt-5 mb-5 p-3 ">
+  <%=link_to user_posts_path(@user) do%>
+    <button class="bg-blue-500 p-3 rounded-lg text-white mx-auto my-auto">See all posts</button>
```

Comment:
- [ ] As stated in the requirements of this project, **See all posts** button should be found in #app/views/users/show.html.erb page. Let's kindly adjust this and make sure that clicking on the link opens the user's post index page.

- [ ] Again, I got the error on the screenshot below when I route to the user's post index page. Please consider fixing it.

![Screenshot from 2023-04-06 12-25-57](https://user-images.githubusercontent.com/63087491/230363720-23ba5dd6-fc85-468d-a3fd-99243dec676a.png)


Please after fixing the error, make sure to render only the first 5 most recent comments and display the rest in the post-show page.

Wish you the best!